scss/_patterns_search-and-filter.scss

```diff
@@ -0,0 +1,168 @@
+@import 'settings';
+
+@mixin vf-p-search-and-filter {
+  @include vf-search-and-filter;
+  @include vf-filter-panel-section;
+}
+
+@mixin vf-search-and-filter {
+  .p-search-and-filter {
+    $height: 2.5rem;
+
+    border: 1px solid $color-mid-dark;
+    position: relative;
+
+    .p-search-and-filter__search-container {
+      align-items: center;
+      background-color: $color-x-light;
+      display: flex;
+      flex-wrap: wrap;
+      height: auto;
+      margin: 0;
+      min-height: $height;
+      overflow: hidden;
+      padding: 0 2rem 0 0.5rem;
+      position: relative;
+
+      &[data-active='true'] {
+        height: auto;
+      }
+
+      &[data-empty='false'],
+      &[aria-expanded='false'] {
+        height: $height;
+      }
+
+      .p-chip {
+        margin-bottom: 0.5rem;
+      }
+
+      .p-search-and-filter__selected-count {
```

Comment:
Another one missing from HTML. 

We should have an example with this state of the component:

<img width="959" alt="Screenshot 2021-03-04 at 16 25 25" src="https://user-images.githubusercontent.com/83575/109986730-4e305c00-7d06-11eb-9fa4-61a55a54c6e0.png">

Reply:
And this? (after clicking on count) and with count in the dropdown:

<img width="470" alt="Screenshot 2021-03-04 at 16 26 51" src="https://user-images.githubusercontent.com/83575/109986970-833cae80-7d06-11eb-9285-4d0e421350ca.png">